scripts/css-vars.ts

```diff
@@ -1,7 +1,7 @@
-import palette from '../src/styles/colors.js'
-import darkPalette from '../src/styles/colors-dark.js'
+import reactComponents from '@safe-global/safe-react-components'
 import spacings from '../src/styles/spacings.js'
 
+const { lightPalette: palette, darkPalette } = reactComponents
```

Comment:
Nit: can we change this to be `lightPalette` as well?

Reply:
changed! thanks